packages/cli/src/commands/init/__tests__/template.test.ts

```diff
@@ -22,7 +19,7 @@ afterEach(() => {
 });
 
 test('installTemplatePackage', async () => {
-  jest.spyOn(PackageManger, 'install').mockImplementationOnce(() => {});
+  jest.spyOn(PackageManger, 'install').mockImplementationOnce(() => null);
```

Comment:
Changed the mock implementation to avoid type conflicts. There was no change in the overall outcome from the unit tests since we're not asserting the result from the mock.